Week1/DailyAssignments/assignment3.js

```diff
@@ -29,13 +27,11 @@ app
   .delete((req, res) => {
     return res.json({ status: "Pending" });
   });
-
 app.post("/api/users", (req, res) => {
   const body = req.body;
   users.push({ ...body, id: users.length + 1 });
   fs.writeFile("./data.json", JSON.stringify(users), (err, data) => {
```

Comment:
put this ./data.json string into a filePath variable so its easy to change from top of file